scripts/categorizer.py

```diff
@@ -385,14 +412,61 @@ def main():
         is_conditional = determine_if_conditional(line_number, multiline_number, line, categorizations) # This might be an exclusive one
         is_function_def = determine_if_function_def(line_number, multiline_number, line, categorizations)
         is_assignment = determine_if_assignment(line_number, multiline_number, line, categorizations, is_function_def)
+        is_function_call = determine_if_function_call(line_number, multiline_number, line, categorizations)
+
+        categorizations.append(LineAndCats(line=line, multiline_statement_number=multilines[line_number],
+                                           conditional=is_conditional, func_def=is_function_def,
+                                           assignment=is_assignment, func_call= is_function_call))
+
+
+        hash_object = hashlib.md5((line + " " + str(line_number)).encode())
+
+        # Create a string which tracks line classfications/categorizations to store in hash
+        line_classifications = ""
+        for number in range(len(LineAndCats._fields)):
+            line_classifications += " " +  "%s " % (LineAndCats._fields[number]) + str(categorizations[-1][number])
+
+        hash_storage[hash_object.hexdigest()] = line_classifications
+
+    # Debug portion -> maybe have all of theese produced into folders when ran???
+    print_all_cats(lines, "func_call", categorizations)
```

Comment:
rename function

Reply:
`print_all_dogs`? ;-)

Reply:
Ha, I love this one! But no, I will probably rename this one to something like print_lines_and_cat.